docs/installation/backend-contribution.md

```diff
@@ -1,25 +1,26 @@
-# Backend installation
+# Installation
 
 <div class="Alert Alert--warning">
 
-This manual is prepared for the developer installation
+This document is for people who want to develop ergonode applications, if you want to use it [check here](installation/backend-development.md)
```

Comment:
This document is for everyone who wants to develop Ergonode application.
If you want to use it